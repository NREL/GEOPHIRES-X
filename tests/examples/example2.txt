GEOPHIRES v2.0 Input File
Created on 6/11/2018
Last modified on 6/12/2018
Geothermal Direct-Use Example Problem using a Linear Heat Sweep Model.

Example 2 Description: This problem considers an EGS reservoir at 3km depth.
Ramey's model is applied to simulate production wellbore heat losses. The heat
is used in direct-use heat application with reinjection temperature of 70deg.C

***Subsurface Technical Parameters***
*************************************

Reservoir Model,2,						---Linear Heat Sweep Model
Reservoir Depth,3,						---[km]
Number of Segments,1,						---[-]
Gradient 1,55,							---[deg.C/km]
Number of Production Wells,2,					---[-]
Number of Injection Wells,2,					---[-]
Production Well Diameter,8,					---[inch]
Injection Well Diameter,8,					---[inch]
Ramey Production Wellbore Model,1,				---Should be 0 (disable) or 1(enable)
Injection Wellbore Temperature Gain,0,				---[deg.C]
Production Flow Rate per Well,30,				---[kg/s]
Fracture Shape,2,						---Should be 1 2 3 or 4. See manual for option details
Fracture Height,300,						---[m]
Reservoir Volume Option,2,					---Should be 1 2 3 or 4. See manual for option details
Fracture Separation,60,						---[m]
Reservoir Impedance,0.2,					---[GPa*s/m3]
Injection Temperature,70,					---[deg.C]
Maximum Drawdown,.3,						---Should be between 0 and 1 depending on redrilling
Reservoir Heat Capacity,975,					---[J/kg/K]
Reservoir Density,3000,						---[kg/m3]
Reservoir Thermal Conductivity,3.2,				---[W/m/K]
Reservoir Porosity,.1,						---[-]

***Surface Technical Parameters***
**********************************
End-Use Option,2,						---Direct Use Heat
Circulation Pump Efficiency,.8,					---[-]
Utilization Factor,.9,						---[-]
End-Use Efficiency Factor,.9,					---[-]
Surface Temperature,15,						---[deg.C]

***Financial Parameters***
**************************

Plant Lifetime,25,						---[years]
Economic Model,2,						---Standard Levelized Cost Model
Discount Rate,.05,						---[-]
Inflation Rate During Construction,0,				---[-]

***Capital and O&M Cost Parameters***
*************************************

Well Drilling and Completion Capital Cost Adjustment Factor,1,	---[M$/well] Use built-in correlations
Well Drilling Cost Correlation,1,				---[-]
Reservoir Stimulation Capital Cost Adjustment Factor,1, 	---[-] Use built-in correlations
Surface Plant Capital Cost Adjustment Factor,1,			---[-] Use built-in correlations
Field Gathering System Capital Cost Adjustment Factor,1,	---[-] Use built-in correlations
Exploration Capital Cost Adjustment Factor,1,			---[-] Use built-in correlations
Wellfield O&M Cost Adjustment Factor,1,				---[-] Use built in correlations
Surface Plant O&M Cost Adjustment Factor,1,			---[-] Use built-in correlations
Water Cost Adjustment Factor,1,					---[-] Use built-in correlations
Electricity Rate,.05,						---[$/kWh]

***Simulation Parameters***
***************************
Print Output to Console,1,					---should be either 0 (do not show) or 1 (show)
Time steps per year,3,						---[-]
