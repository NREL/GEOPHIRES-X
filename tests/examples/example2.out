                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.0
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-02-13
 Simulation Time:  09:44
 Calculation Time:      0.215 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Direct-Use Heat
      Average Direct-Use Heat Production:                    21.03 MW
      Direct-Use heat breakeven price:                        7.23 USD/MMBTU
      Number of production wells:                             2
      Number of injection wells:                              2
      Flowrate per production well:                          30.0 kg/sec
      Well depth (or total length, if not vertical):          3.0 kilometer
      Geothermal gradient:                                    0.0550 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model = Standard Levelized Cost
      Interest Rate:                                          5.00
      Accrued financing during construction:                  0.00
      Project lifetime:                                      25 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                          -1.73 MUSD
      Project IRR:                                           0.06 %
      Project VIR=PI=PIR:                                    0.96
      Project MOIC:                                          0.51

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             2
      Number of Injection Wells:                              2
      Well depth (or total length, if not vertical):          3.0 kilometer
      Water loss rate:                                        0.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 70.0 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               6.7 degC
      Flowrate per production well:                          30.0 kg/sec
      Injection well casing ID:                               8.000 in
      Production well casing ID:                              8.000 in
      Number of times redrilling:                             0


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0550 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = 1-D Linear Heat Sweep Model
      Bottom-hole temperature:                              180.00 degC
      Fracture model = Circular fracture with known diameter
      Well seperation: fracture diameter:                   300.00 meter
      Fracture area:                                      70685.83 m**2
      Number of fractures calculated with reservoir volume and fracture separation as input
      Number of fractures:                                   30.47
      Fracture separation:                                   60.00 meter
      Reservoir volume:                               125000000 m**3
      Reservoir impedance:                                    0.20 GPa.s/m**3
      Reservoir density:                                   3000.00 kg/m**3
      Reservoir thermal conductivity:                         3.20 watt/m/kelvin
      Reservoir heat capacity:                              975.00 J/kg/kelvin
      Reservoir porosity:                                    10.00


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       172.6 degC
      Average Production Temperature:                       165.7 degC
      Minimum Production Temperature:                       154.8 degC
      Initial Production Temperature:                       169.6 degC
      Average Reservoir Heat Extraction:                     23.36 MW
      Production Wellbore Heat Transmission Model = Ramey Model
      Average Production Well Temperature Drop:               6.7 degC
      Total Average Pressure Drop:                        10383.3 kPa
      Average Injection Well Pressure Drop:                 110.6 kPa
      Average Reservoir Pressure Drop:                    12215.4 kPa
      Average Production Well Pressure Drop:                114.7 kPa
      Average Buoyancy Pressure Drop:                     -2057.4 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      21.95 MUSD
         Drilling and completion costs per well:              5.49 MUSD
         Stimulation costs:                                   3.02 MUSD
         Surface power plant costs:                           8.07 MUSD
         Field gathering system costs:                        2.55 MUSD
         Total surface equipment costs:                      10.62 MUSD
         Exploration costs:                                   5.33 MUSD
      Total capital costs:                                   40.91 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.42 MUSD/yr
         Power plant maintenance costs:                       0.65 MUSD/yr
         Water costs:                                         0.00 MUSD/yr
         Average Reservoir Pumping Cost:                      0.30 MUSD/yr
      Total operating and maintenance costs:                  1.37 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Maximum Net Heat Production:                           22.55 MW
      Average Net Heat Production:                           21.03 MW
      Minimum Net Heat Production:                           18.64 MW
      Initial Net Heat Production:                           21.89 MW
      Average Annual Heat Production:                       163.65 GWh
      Average Pumping Power:                                  0.77 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET
             DRAWDOWN             TEMPERATURE             POWER              HEAT
                                   (deg C)                (MW)               (MW)
   0           1.0000                169.64               0.7567           21.8934
   1           1.0075                170.91               0.7561           22.1720
   2           1.0113                171.56               0.7558           22.3152
   3           1.0133                171.90               0.7556           22.3898
   4           1.0146                172.13               0.7555           22.4392
   5           1.0156                172.29               0.7555           22.4757
   6           1.0164                172.42               0.7554           22.5043
   7           1.0170                172.53               0.7554           22.5278
   8           1.0176                172.62               0.7553           22.5477
   9           0.9956                168.90               0.7627           21.7304
  10           0.9916                168.22               0.7642           21.5798
  11           0.9873                167.49               0.7656           21.4207
  12           0.9828                166.73               0.7672           21.2536
  13           0.9782                165.94               0.7688           21.0793
  14           0.9733                165.11               0.7704           20.8980
  15           0.9683                164.26               0.7721           20.7103
  16           0.9631                163.38               0.7739           20.5167
  17           0.9577                162.47               0.7756           20.3174
  18           0.9522                161.54               0.7774           20.1129
  19           0.9466                160.59               0.7793           19.9035
  20           0.9409                159.61               0.7812           19.6896
  21           0.9350                158.62               0.7831           19.4715
  22           0.9291                157.61               0.7850           19.2495
  23           0.9230                156.58               0.7869           19.0239
  24           0.9169                155.54               0.7889           18.7951


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR               HEAT                       HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                 173.5                       192.7                 39.52                   1.73
   2                 175.4                       194.9                 38.82                   3.47
   3                 176.2                       195.8                 38.12                   5.22
   4                 176.7                       196.4                 37.41                   6.98
   5                 177.1                       196.7                 36.70                   8.74
   6                 177.3                       197.0                 35.99                  10.50
   7                 177.5                       197.2                 35.28                  12.27
   8                 177.7                       197.4                 34.57                  14.04
   9                 172.8                       192.0                 33.88                  15.76
  10                 170.7                       189.7                 33.20                  17.45
  11                 169.5                       188.3                 32.52                  19.14
  12                 168.2                       186.9                 31.85                  20.81
  13                 166.9                       185.4                 31.18                  22.47
  14                 165.5                       183.9                 30.52                  24.12
  15                 164.0                       182.2                 29.86                  25.75
  16                 162.5                       180.6                 29.21                  27.37
  17                 161.0                       178.9                 28.57                  28.97
  18                 159.4                       177.1                 27.93                  30.55
  19                 157.7                       175.3                 27.30                  32.12
  20                 156.1                       173.4                 26.68                  33.67
  21                 154.4                       171.5                 26.06                  35.21
  22                 152.6                       169.6                 25.45                  36.73
  23                 150.9                       167.6                 24.84                  38.23
  24                 149.1                       165.6                 24.25                  39.71
  25                  98.4                       109.3                 23.85                  40.69
